<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>HtmlViewer</class>
    <widget class="QMainWindow" name="HtmlViewer">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>997</width>
                <height>632</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>MainWindow</string>
        </property>
        <widget class="QWidget" name="centralwidget">
            <layout class="QGridLayout" name="gridLayout">
                <item row="1" column="0" colspan="3">
                    <widget class="QWebEngineView" name="webView" native="true">
                        <property name="url" stdset="0">
                            <url>
                                <string>about:blank</string>
                            </url>
                        </property>
                    </widget>
                </item>
                <item row="0" column="0">
                    <widget class="QPushButton" name="prevPageButton">
                        <property name="minimumSize">
                            <size>
                                <width>75</width>
                                <height>0</height>
                            </size>
                        </property>
                        <property name="maximumSize">
                            <size>
                                <width>75</width>
                                <height>16777215</height>
                            </size>
                        </property>
                        <property name="text">
                            <string>&lt;</string>
                        </property>
                    </widget>
                </item>
                <item row="0" column="1">
                    <widget class="QPushButton" name="nextPageButton">
                        <property name="minimumSize">
                            <size>
                                <width>75</width>
                                <height>0</height>
                            </size>
                        </property>
                        <property name="maximumSize">
                            <size>
                                <width>75</width>
                                <height>16777215</height>
                            </size>
                        </property>
                        <property name="text">
                            <string>&gt;</string>
                        </property>
                    </widget>
                </item>
            </layout>
        </widget>
        <widget class="QMenuBar" name="menubar">
            <property name="geometry">
                <rect>
                    <x>0</x>
                    <y>0</y>
                    <width>997</width>
                    <height>20</height>
                </rect>
            </property>
            <widget class="QMenu" name="settingsMenu">
                <property name="title">
                    <string>Настройки</string>
                </property>
                <addaction name="settingsMenuBar"/>
            </widget>
            <addaction name="settingsMenu"/>
        </widget>
        <widget class="QStatusBar" name="statusbar"/>
        <action name="settingsMenuBar">
            <property name="text">
                <string>Настройки</string>
            </property>
        </action>
    </widget>
    <customwidgets>
        <customwidget>
            <class>QWebEngineView</class>
            <extends>QWidget</extends>
            <header location="global">QtWebEngineWidgets/QWebEngineView</header>
        </customwidget>
    </customwidgets>
    <resources/>
    <connections/>
</ui>
